data_validation/combiner.py

```diff
@@ -405,3 +431,45 @@ def _add_metadata(joined, run_metadata):
     ]
 
     return joined
+
+
+def get_summary(joined_df: "DataFrame", source_df: "DataFrame", target_df: "DataFrame"):
```

Comment:
Tiny nitpick, feel free to ignore.

As `get_summary` is private to this module I would prefix the name an underscore,  `_get_summary`